recipes/backup_diy.rb

```diff
@@ -0,0 +1,49 @@
+Stash.check_for_old_attributes!(node)
+
+settings = Stash.settings(node)
+
+package 'jq'
```

Comment:
Yes, you're right. `jq` is in EPEL repo only. What do you think about adding this line here?
```ruby
include_recipe 'yum-epel' if node['platform_family'] == 'rhel'
```
P.s. And add `yum-epel` dependensy to `metadata.rb`